tests/buildtest/crates/lockfile-collision/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "lockfile-collision"
+version = "0.1.0"
+authors = ["Pietro Albini <pietro@pietroalbini.org>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+lockfile-collision = { path = "./crates/lockfile-collision" }
```

Comment:
can you explain what the error is?

Reply:
The package name of this package and its dependency are the same.


I think this is based on [this error](https://crater-reports.s3.amazonaws.com/beta-1.86-1/1.85.0/gh/shadowmint.loghaul/log.txt)

```
[INFO] [stderr]     Blocking waiting for file lock on package cache
[INFO] [stderr] error: package collision in the lockfile: packages loghaul v0.1.0 (/workspace/builds/worker-3-tc2/source) and loghaul v0.1.0 (/workspace/builds/worker-3-tc2/source/crates/loghaul) are different, but only one can be written to lockfile unambiguously
```